resources/privacy-and-gdpr.md

```diff
@@ -1,16 +1,16 @@
 # Privacy and GDPR
```

Comment:
Can we still update the title, even if we can't change the file name yet? Maybe something like "Privacy and consent"?